portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,119 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+      <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+      integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+      crossorigin="anonymous"
+      />
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        
+      <header>
+          <div class="header-menu">
+              <a href="#">Skills</a>
+              <a href="#">Portfolio</a>
+              <a href="#">Education</a>
+              <a href="#">Videos</a>
+          </div>
+      </header>
+      <h1 class="title-header">Joachim Isaac's Portfolio!</h1>
+      <div class="img-container"><img src="/images/joachim_isaac.png"></div>
+      
+      <p class="introduction">
+        My name is Joachim Isaac, I am a Software engineer by trade, an avid amateur boxer, weightlifter and stock trader.
+        I am very passionate about technology - not a specific framework or language.
+        I'm interested in web developement and artificial intellegence to create a cool web applications, that make
+        people's lives easier.
+      </p>
+   
+     
+      <p class="indicator">Click Here to find out one of my interests</p>
+      <div class="button-container"><button onclick="addRandomGreeting()">Click</button></div>
```

Comment:
Inline event handlers like this onclick="..." can be done for simplicity, but you'll also want to learn how to add them directly from scripts.  There's no need to change this now, but take a look at the following article when you have time: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events